app/components/data-menu/layer-directives/eventseries-directive.js

```diff
@@ -26,24 +26,24 @@ angular.module('data-menu')
           MapService.mapLayers.push(eventseriesMapLayer({
             color: response.color,
             uuid: scope.layer.uuid,
-            url: 'api/v2/events/?event_series=' + scope.layer.uuid,
+            url: 'api/v3/events/?event_series=' + scope.layer.uuid,
             spatialSelect: MapService.spatialSelect,
             zIndex: LayerAdderService.getZIndex(scope.layer)
           }));
 
           DataService.dataLayers.push(eventseriesDataLayer({
             uuid: scope.layer.uuid,
-            url: 'api/v2/events/?event_series=' + scope.layer.uuid,
+            url: 'api/v3/events/?event_series=' + scope.layer.uuid,
             agg: response.aggregation_type,
             format: 'Vector',
             color: response.color,
             type: "eventseries",
             scale: response.observation_type
               && response.observation_type.scale,
             quantity: response.observation_type
-              && response.observation_type.parameter_short_display_name,
+              && response.observation_type.parameter,
             unit: response.observation_type
-              && response.observation_type.referenced_unit_short_display_name
+              && response.observation_type.referenced_unit
```

Comment:
@RoelvandenBerg : bestaat "referenced_unit" wel? Deze is niet gedefinieerd in de RAML en ik zie hem ook niet in de API response: https://demo.lizard.net/api/v3/observationtypes/.